Packs/SIGNL4/Integrations/SIGNL4/SIGNL4.yml

```diff
@@ -0,0 +1,53 @@
+category: Messaging and Conferencing
+commonfields:
+  id: SIGNL4
+  version: -1
+configuration:
+- display: Team or Integration Secret
+  name: secret
+  hiddenusername: true
+  type: 4
+  required: true
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+description: SIGNL4, mobile alerting.
+display: SIGNL4
+name: SIGNL4
+script:
+  commands:
+  - arguments:
+    - description: The alert title.
+      name: title
+      required: true
+    - description: The alert text.
+      name: message
+      required: true
+    - description: The external ID is used to reference the alert in order to close it later.
+      name: s4_external_id
```

Comment:
Up to you, but I don't think the s4 prefix is ​​needed for the arguments.

Reply:
If OK, I would leave it to tell users that this is the X-S4-ExternalID they might already know.